db.sql

```diff
@@ -99,3 +99,13 @@ alter table shop
     add ignores text default '[]' null after last_scraped_error;
 
 ALTER TABLE `shop` ADD COLUMN `shop_image` VARCHAR(128) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci NULL DEFAULT NULL AFTER `ignores`;
+
+CREATE TABLE `shop_changelog` (
+    `shop_id` int NOT NULL,

```

Comment:
that table needs an id if you have multiple records + an primary key

Reply:
Oh, I had added it on the dev mysql server but forget it in the db.sql